bsyncviewer/templates/auditors.html

```diff
@@ -14,15 +14,15 @@
 
         <div class="row" style="padding-bottom: 7em; padding-top: 5em">
             <div class="col-lg-6">
-                <h4>Direct integration with the Audit Template Tool (ATT).</h4>
+                <h4>Direct integration with the Audit Template (AT).</h4>
```

Comment:
I would remove "the" from this sentence and just say "direct integration with Audit Template"